synapseformation/create.py

```diff
@@ -271,7 +296,7 @@ def get_or_create_challenge(self, **kwargs) -> dict:
             challenge = self._create_challenge(**kwargs)
         except SynapseHTTPError as err:
             # Must check for 409 error
```

Comment:
I myself don't understand why the 400 error is returned, but for whatever reason a 400 error is returned when I try to create a challenge object that already exists.  We could file a JIRA to PFLM to ask why a 400 is returned instead of 409.

Reply:
https://sagebionetworks.jira.com/browse/PLFM-6454